src/sentry/spans/consumers/process/factory.py

```diff
@@ -86,6 +90,33 @@ def _build_snuba_span(relay_span: Mapping[str, Any]) -> MutableMapping[str, Any]
 
     snuba_span["sentry_tags"] = sentry_tags
 
+    grouping_config_id = {"id": DEFAULT_CONFIG_ID}
+    grouping_config = load_span_grouping_config(grouping_config_id)
```

Comment:
You could also do
```suggestion
    grouping_config = load_span_grouping_config()
```

which will load the default config, but not a big deal at all